sass/base/_base.scss

```diff
@@ -0,0 +1,27 @@
+html, a {
+  color: text("primary");
+}
+
+h1, h2 {
+  font-family: $secondary-font;
+}
+
+h1 {
+
+  [data-section="app-reviews"] &,
+  [data-section="more-apps"] &,
+  [data-section="appbox-screenshots"] & {
+    text-align: center;
+  }
```

Comment:
The `&` used after a nested selector inverts the scoping/context/meaning of the block. I discourage it's use in this manner. This is a style that should be applied from the context of the respective components (ie: `[data-section="app-reviews"] h1 {...}`

Also, see previous comments on these selectors.